app/src/main/java/com/simply/birthdayapp/presentation/ui/components/RoundAsyncImage.kt

```diff
@@ -0,0 +1,53 @@
+package com.simply.birthdayapp.presentation.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import coil.size.Size
+
+@Composable
+fun RoundAsyncImage(
+    modifier: Modifier = Modifier,
+    data: Any? = null,
+    borderColor: Color,
+    borderWidth: Dp,
```

Comment:
You can give default values for `borderColor` and `borderWidth` too

Reply:
[Given](https://github.com/SimplyTechnologies/android-internship/pull/14/commits/21b292e2c7e5a0b284d601253c8c8ae604b63523)